service-module/src/main/java/ru/mail/dimaushenko/service/ConvertService.java

```diff
@@ -0,0 +1,43 @@
+package ru.mail.dimaushenko.service;
+
+import java.util.List;
+import ru.mail.dimaushenko.repository.model.Item;
+import ru.mail.dimaushenko.repository.model.ItemDetails;
+import ru.mail.dimaushenko.repository.model.Shop;
+import ru.mail.dimaushenko.repository.model.ShopWithItem;
+import ru.mail.dimaushenko.service.model.AddItemDTO;
+import ru.mail.dimaushenko.service.model.ItemDTO;
+import ru.mail.dimaushenko.service.model.ItemDetailsDTO;
+import ru.mail.dimaushenko.service.model.ShopDTO;
+import ru.mail.dimaushenko.service.model.ShopWithItemDTO;
+import ru.mail.dimaushenko.service.model.addShopDTO;
+
+public interface ConvertService {
```

Comment:
Better to use separate convertors(as Component) for separate domains with generic interface